addons/survey/views/survey_templates.xml

```diff
@@ -318,9 +316,9 @@
         <t t-set="default_constr_error_msg">This question requires an answer.</t>
         <t t-set="default_validation_error_msg">The answer you entered is not valid.</t>
         <t t-set="default_comments_message">If other, please specify:</t>
-        <div t-att-class="'js_question-wrapper pb-4 %s %s' % ('d-none' if not display_question else '', 'me-2' if extra_right_margin else '')"
+        <div t-attf-class="js_question-wrapper pb-4 #{'d-none' if not display_question else ''} #{'me-2' if extra_right_margin else ''} #{'w-lg-50 mx-lg-auto' if survey.questions_layout != 'page_per_question' else ''}"
```

Comment:
This line should probably be wrapped at some point(s);

Could we please  also take this opportunity to clean lines 330-337 into something like this?
```xml
<t t-if="..." t-call="..."/>
<t t-elif="..." t-call="..."/>
...
```

Reply:
> This line should probably be wrapped at some point(s);

I did not get you. can explain it in detail?